velox/exec/OperatorUtils.cpp

```diff
@@ -118,22 +118,24 @@ void gatherCopy(
 }
 } // namespace
 
-void deselectRowsWithNulls(
+bool deselectRowsWithNulls(
     const std::vector<std::unique_ptr<VectorHasher>>& hashers,
     SelectivityVector& rows) {
-  bool anyChange = false;
+  bool foundNull = false;
   for (int32_t i = 0; i < hashers.size(); ++i) {
     auto& decoded = hashers[i]->decodedVector();
     if (decoded.mayHaveNulls()) {
-      anyChange = true;
+      foundNull = true;
```

Comment:
Do we need to check if the bits are not all set?

Reply:
That's a good catch. Will update.